`default-features = false` does not build

Was trying to use `default-features = false` in one of my projects for termimad and I seem to get this error:

```
error[E0609]: no field `composite` on type `&composite::FmtComposite<'_>`
   --> /Users/strawberry/.cargo/registry/src/index.crates.io-6f17d22bba15001f/termimad-0.29.3/src/skin.rs:565:22
    |
565 |         for c in &fc.composite.compounds {
    |                      ^^^^^^^^^ unknown field
    |
    = note: available fields are: `kind`, `compounds`, `visible_length`, `spacing`

For more information about this error, try `rustc --explain E0609`.
error: could not compile `termimad` (lib) due to 1 previous error
```

--- a/bacon.toml
+++ b/bacon.toml
@@ -9,6 +9,10 @@ default_job = "check-all"
 command = ["cargo", "check", "--color", "always"]
 need_stdout = false
 
+[jobs.check-no-features]
+command = ["cargo", "check", "--no-default-features", "--color", "always"]
+need_stdout = false
+
 [jobs.check-all]
 command = ["cargo", "check", "--all-targets", "--color", "always"]
 need_stdout = false

--- a/src/skin.rs
+++ b/src/skin.rs
@@ -24,7 +24,6 @@ use {
         MAX_HEADER_DEPTH,
     },
     std::{
-        collections::HashMap,
         fmt,
         io::Write,
     },
@@ -57,7 +56,7 @@ pub struct MadSkin {
     /// (comments welcome)
     /// Do not use compounds with a length different than 1.
     #[cfg(feature = "special-renders")]
-    pub special_chars: HashMap<Compound<'static>, StyledChar>,
+    pub special_chars: std::collections::HashMap<Compound<'static>, StyledChar>,
 
 }
 
@@ -92,7 +91,7 @@ impl Default for MadSkin {
             list_items_indentation_mode: Default::default(),
 
             #[cfg(feature = "special-renders")]
-            special_chars: HashMap::new(),
+            special_chars: std::collections::HashMap::new(),
         };
         skin.code_block.set_fgbg(gray(17), gray(3));
         for h in &mut skin.headers {
@@ -128,7 +127,7 @@ impl MadSkin {
             ellipsis: CompoundStyle::default(),
             list_items_indentation_mode: Default::default(),
             #[cfg(feature = "special-renders")]
-            special_chars: HashMap::new(),
+            special_chars: std::collections::HashMap::new(),
             table_border_chars: STANDARD_TABLE_BORDER_CHARS,
         }
     }
@@ -562,7 +561,7 @@ impl MadSkin {
             }
         }
         #[cfg(not(feature = "special-renders"))]
-        for c in &fc.composite.compounds {
+        for c in &fc.compounds {
             let os = self.compound_style(ls, c);
             write!(f, "{}", os.apply_to(c.as_str()))?;
         }

